src/elements/input/src/styles.ts

```diff
@@ -4,20 +4,31 @@ import { SxStyleProp } from '../../../../types/theme-ui'
 
 const { tomato, UswitchNavy } = colors
 
+const boxShadow = (theme: any, hasError: boolean, hasFocus: boolean) => {
+  if (hasError) {
+    return (
+      theme.elements.input?.error?.boxShadow ||
+      `inset 0 0 0 1px
+     ${theme.colors[theme.elements.input?.error?.color] ?? tomato}`
+    )
+  } else if (hasFocus) {
+    return (
+      theme.elements.input?.focus?.boxShadow ||
+      `inset 0 0 0 1px ${theme.colors[theme.elements.input?.focus?.color] ??
+        UswitchNavy}`
+    )
+  } else {
+    return 'inset 0 2px 5px 0 rgba(0, 0, 0, 0.1)'
```

Comment:
Should this not come from the theme too?

Reply:
oh, yeah, you're right, forgot